src/OpenTelemetry.AutoInstrumentation.Native/continuous_profiler.cpp

```diff
@@ -218,20 +218,17 @@ int32_t SelectiveSamplingConsumeAndReplaceBuffer(int32_t len, unsigned char* buf
         trace::Logger::Warn("Unexpected 0/null buffer to SelectiveSamplingConsumeAndReplaceBuffer");
```

Comment:
As per new implementation the name could be revised - 
SelectiveSamplingConsumeAndClearBuffer

Reply:
Renamed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/4321/commits/134fe29262766d35c7f0dbc409799468c213a6ab